docs/rfcs/0002_templates_mvp.md

```diff
@@ -0,0 +1,359 @@
+| description   | text        | A short blurb describing the entity the submission template represents                                                                                                                             |
+| fields_schema | JSON        | The template submission rules. A JSON blob with a standard structure which will encode the template field objects having things like: label, key, description, type, values_accepted, instructions |
+| created_at    | datetime    | Timestamp of creation                                                                                                                                                                              |
+| modified_at   | varchar     | Timestamp of update                                                                                                                                                                                |
+| creator       | foreign key | Foreign key to User that created the submission template                                                                                                                                           |
+| organization  | foreign key | Foreign key to Organization that owns the template package                                                                                                                                         |
+
+#### Django Model Schema
+
+| Name                          | Type           | Description                                                                                                          |
+| ----------------------------- | -------------- | -------------------------------------------------------------------------------------------------------------------- |
+| All fields in DB table schema |
+| released                      | BooleanField   | A computed field (not stored in db) denoting whether the template has been released in one or more template packages |
+| submission_template_file      | FileField      | Reference to the actual template file in a storage system                                                            |
+| extract_config_file           | FileField      | Reference to the actual extract config file in a storage system                                                      |
+| template_packages             | RelatedManager | List of template packages this template is contained in                                                              |
+
+### TemplatePackage
+
+A template package is a set of submission templates that will be used to set
+submission standards and collect data for a study.
+
+The concept of a template package is useful for several reasons. First,
+templates are typically used in conjunction with one and other because a set of
+templates usually represents one data model, and each template typically
+captures data for a single entity in the model.
```

Comment:
I'm not sure what to suggest here because I don't really understand what you're trying to say. I assume `with one and other` is supposed to be `with one another`, but I'm still not entirely sure what you mean.